docs/tutorials/kubernetes/efficient-data-sharing-in-kubeflow-with-vineyard-csi-driver.rst

```diff
@@ -248,11 +257,11 @@ We delete the time of init data loading, and the results are as follows:
 +------------+------------------+---------------+
 | data scale | without vineyard | with vineyard |
 +============+==================+===============+
-| 8500 Mi    | 36.7s            | 0.02s         |
+| 8500 Mi    | 37.3s            | 0.04s         |
 +------------+------------------+---------------+
-| 12000 Mi   | 45.7s            | 0.02s         |
+| 12000 Mi   | 49.5s            | 0.04s         |
 +------------+------------------+---------------+
-| 15000 Mi   | 128.6s           | 0.04s         |
+| 15000 Mi   | 61.7s            | 0.04s         |
 +------------+------------------+---------------+
```

Comment:
Why this PR affects the performances of "without vineyard" case?

Reply:
The previous time is wrong as it is mixed with the initial reading time. My bad.